C/COSMA/build_tarballs.jl

```diff
@@ -0,0 +1,76 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "COSMA"
+version = v"2.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/eth-cscs/COSMA/releases/download/v2.2.0/cosma.tar.gz", "1eb92a98110df595070a12193b9221eecf9d103ced8836c960f6c79a2bd553ca")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+if [[ "$nbits" == "64" ]] && [[ "$target" != aarch64-* ]]; then
+  BLAS_LAPACK_LIB="$libdir/libopenblas64_.$dlext"
+else
+  BLAS_LAPACK_LIB="$libdir/libopenblas.$dlext"
+fi
+
+mkdir build
+cd build
+
+cmake ../cosma \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCOSMA_WITH_TESTS=OFF \
+    -DCOSMA_WITH_APPS=OFF \
+    -DCOSMA_WITH_BENCHMARKS=OFF \
+    -DCOSMA_WITH_INSTALL=ON \
+    -DCOSMA_WITH_PROFILING=OFF \
+    -DBUILD_SHARED_LIBS=ON \
+    -DCOSMA_BLAS=OPENBLAS \
+    -DCOSMA_SCALAPACK=OFF \
+    -DMPI_CXX_COMPILER=$bindir/mpicxx \
+    -DMPI_C_COMPILER=$bindir/mpicc \
+    -DOPENBLAS_LIBRARIES=$BLAS_LAPACK_LIB \
+    -DOPENBLAS_INCLUDE_DIR=$libdir/../include
```

Comment:
```suggestion
    -DOPENBLAS_INCLUDE_DIR="${includedir}"
```

Reply:
```# echo "$includedir"

```
?

Reply:
You're using an old version of BinaryBuilder: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/765 :wink: